test/topic.e2e-spec.ts

```diff
@@ -179,18 +192,12 @@ describe('Topic Module', () => {
       expect(respond2.status).toBe(200);
       expect(respond2.body.code).toBe(200);
       expect(respond2.body.data.topics.length).toBe(3);
-      expect(respond2.body.data.topics[0].name).toBe(
-        `${TestTopicPrefix} 高等数学`,
-      );
-      expect(respond2.body.data.topics[1].name).toBe(
-        `${TestTopicPrefix} 高等代数`,
-      );
-      expect(respond2.body.data.topics[2].name).toBe(
-        `${TestTopicPrefix} 高等数学习题`,
-      );
-      expect(respond2.body.data.page.page_start).toBe(
-        respond2.body.data.topics[0].id,
-      );
+      expect(respond2.body.data.topics[0].name).toContain(TestTopicCode);
```

Comment:
why request x3 in this it block

Reply:
I mean, why not break it up into three `it`s?

Reply:
because the asserts use the respond of other requests